### PR TITLE
Batch withdraw

### PR DESCRIPTION
**Pull request by lawlawlaw**
* Enable batch withdrawal by selecting multiple coins in the GUI
* Coin selection is toggled by repeated clicks on the coin listing in the Swap and Withdrawal page. Multiple coins can be selected simultaneously by clicking on each of the required coins.
* Requires mercury server batch_withdraw pull request: https://github.com/commerceblock/mercury/pull/160

### CODE REVIEW

File: src/components/coins/coins.js [hunk 1]
```diff
@@ -40,7 +40,13 @@ const INITIAL_SORT_BY = {
 	by: 'value'
 };
 
+function useForceUpdate(){
```
Comment (86667) [inline src/components/coins/coins.js:43]: there is already a way to refresh Coins. Check how `props.refresh` is used and accessed by other Pages.

File: src/components/panelCoinsActivity/panelCoinsActivity.js [hunk 2]
```diff
@@ -7,7 +7,21 @@ import './panelCoinsActivity.css';
 import '../index.css';
 
 const PanelCoinsActivity = (props) => {
-    const [selectedCoin, setSelectedCoin] = useState(null); // store selected coins shared_key_id
+    const [selectedCoins, setSelectedCoins] = useState([]); // store selected coins shared_key_id
+
+    function setSelectedCoin(statechain_id) {
```
Comment (86667) [inline src/components/panelCoinsActivity/panelCoinsActivity.js:12]: Lets keep function syntax to be consistent throughout the codebase. 

`const setSelectedCoin = () => {}`

File: src/containers/Send/Send.js [hunk 3]
```diff
@@ -16,7 +16,21 @@ const SendStatecoinPage = () => {
   const dispatch = useDispatch();
   const balance_info = useSelector(state => state.walletData).balance_info;
 
-  const [selectedCoin, setSelectedCoin] = useState(null); // store selected coins shared_key_id
+  const [selectedCoins, setSelectedCoins] = useState([]); // store selected coins shared_key_id
+  function setSelectedCoin(statechain_id) {
+    setSelectedCoins(
+        prevSelectedCoins => {
```
Comment (86667) [inline src/containers/Send/Send.js:22]: Just set selected coin to `statechain_id`? 

```
setSelectedCoins([statechain_id]).
```

Im not sure if Send page should care about the data structures of Coins panel. It only cares about the single selectedCoin. I think it confuses the code to have setSelectedCoins([]) with an array on this page.

File: src/settings.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "min_anon_set": 3,
-  "state_entity_endpoint": "https://beta.mercurywallet.io",
-  "swap_conductor_endpoint": "https://beta.mercurywallet.io",
+  "state_entity_endpoint": "http://0.0.0.0:8000",
```
Comment (86667) [inline src/settings.json:3]: revert

**Review by 86667 — CHANGES_REQUESTED** [review]

File: src/components/panelCoinsActivity/panelCoinsActivity.js (hunk 2 above)
Comment (lawlawlaw) [reply src/components/panelCoinsActivity/panelCoinsActivity.js]: done

File: src/settings.json (hunk 4 above)
Comment (lawlawlaw) [reply src/settings.json]: done

File: src/containers/Send/Send.js (hunk 3 above)
Comment (lawlawlaw) [reply src/containers/Send/Send.js]: done

**Comment by 86667** [comment]
Thanks for changes. 

Looks good apart from in removig coin from Swap i get 

```
TypeError: Cannot read property 'statechain_id' of undefined

**Review by 86667 — APPROVED** [review]

**Pull request closed without merging**